### PR TITLE
Options to omit labels in K8s pod logs

### PR DESCRIPTION
**Pull request by alesnovak-s1**
From documentation:
| `k8s_label_include_globs`                     | Optional, (defaults to ['*']). Specifies a list of K8s labels to be added to logs. | 
| `k8s_label_exclude_globs`                     | Optional, (defaults to []]). Specifies a list of K8s labels to be ignored and not added to logs. |

Follows the same logic as label_include_globs and label_exclude_globs from the docker_monitor.

Implements:
https://sentinelone.atlassian.net/browse/DTIN-3678

As part of this PR I've added some multiaccount documentation from the generated md file to the kubernetes_monitor.py mondule.

### CODE REVIEW

File: scalyr_agent/builtin_monitors/kubernetes_monitor.py [hunk 1]
```diff
@@ -3321,7 +3349,11 @@ def __get_log_config_for_container(self, cid, info, k8s_cache, base_attributes):
                         # field `_k8s_ck`
                         container_attributes["_k8s_dn"] = controller.name
                         # `_k8s_dl` is translated to `k8s-labels`
-                        container_attributes["_k8s_dl"] = controller.flat_labels
```
Comment (jmakar-s1) [inline scalyr_agent/builtin_monitors/kubernetes_monitor.py:3324]: This is the only place flat_labels is used, recommend removing it from the Controller constructor

File: scripts/cicd/common/scalyr_query.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+import requests
+import time
+import os
+import sys
+
+
+def scalyr_query(token: str, filter: str, time_start: str, retries: int):
+    def post_request():
+        return requests.post(
```
Comment (jmakar-s1) [inline scripts/cicd/common/scalyr_query.py:12]: Should add a timeout when using requests, the default it to wait forever

**Review by jmakar-s1 — APPROVED** [review]
LGTM, just had one minor comment

**Comment by alesnovak-s1** [comment]
Resolves #1213

**Pull request merged**